backup.sh

```diff
@@ -26,6 +26,14 @@ docker exec djangoapp-${env} /bin/sh -c "tar -C $mediaDirectoryInContainer -cz -
 
 printf "Created a new backup in %s \n" "$directoryOfBackup"/"$date"
 
+# Save logs
+if [[ "${env}" = "production" ]]; then
+    directoryOfLogs="$directoryOfBackup"/production-logs/"$date"
```

Comment:
I would change the path to this so that all data from the same backup task is contained in a common folder instead of in two parallel ones:
```suggestion
    directoryOfLogs="$directoryOfBackup"/"$date"/production-logs
```

Reply:
That should also take care of automatically deleting the old logs, since they would then be stored in the same folder as the existing backup and would be deleted together by the lines below.

Reply:
That was precisely the reason, why I chose another directory: We want to preserve the logs in order to be able to retrieve visitor counts. I think, the backup is done every day and 7 backups are kept. The logs would be thrown away after 7 days...

Reply:
Is there a time after which we could tolerate throwing them away? Writing lots of logs to the filesystem without any mechanism to limit the size is asking for problems imho. Since this is supposed to be a temporary solution, I'm okay with not rotating the files for now, as long as we don't write all logs, but only the log lines needed for the usage statistics (or even calculate them immediately) to reduce the risk of filling the disk until the rral solution exists.